R/utils-misc.R

```diff
@@ -33,28 +33,26 @@ check_pname <- function(res, preds, object, newname = NULL) {
 
 
 grab_forged_keys <- function(forged, workflow, new_data) {
-  keys <- c("geo_value", "time_value", "key")
   forged_roles <- names(forged$extras$roles)
-  extras <- dplyr::bind_cols(forged$extras$roles[forged_roles %in% keys])
+  extras <- dplyr::bind_cols(forged$extras$roles[forged_roles %in% c("geo_value", "time_value", "key")])
```

Comment:
preference?

Reply:
Minor preference: helped me to be explicit rather than keys[1:2] and new_keys[-c(1:2)] downstream when first trying to understand the function.

Reply:
I definitely don't like the `keys[1:2]` thing.